R/report_results.R

```diff
@@ -161,16 +162,16 @@ report_results<-function(kmer_hist,kmer_hist_orig, k, p, container, foldername,
   cex.lab=font_size, cex.axis=font_size, cex.main=font_size, cex.sub=font_size)
   rect(1e-10, 1e-10, max(kmer_hist_transform[,1])*10 , max(kmer_hist_transform[,2])*10, col=COLOR_BGCOLOR)
   points(kmer_hist_transform, type="h", col=COLOR_HIST, lwd=2)
-  if(length(kmer_hist[,1])!=length(kmer_hist_transform[,1])){
+  if( ISCROPPED ){
     abline(v=length(kmer_hist[,1]),col=COLOR_COVTHRES,lty="dashed", lwd=3)
   }
   box(col="black")
 
 
   if(!is.null(model))
   {
-    x=kmer_hist[[1]]
-    y=kmer_hist[[2]]
+    x=kmer_hist_orig[[1]]
+    y=kmer_hist_orig[[2]]
```

Comment:
why are you switching to the original dataset here? Unless Im missing something, this will have the artificially inflated counts in the last position that could confuse the model fitting

Reply:
This section of code gets excuted only after the model is fit, but before some of the genomic features get computed. Namely - genome size (https://github.com/tbenavi1/genomescope2.0/blob/master/R/report_results.R#L240-L248)

I tried to have GenomeScope ignore the last position all the way till the genome size estimate, when it needs to be added back for FastK to work.

Reply:
right of course!